### ISSUE TITLE
FE: Implement the settings design from Figma

### ISSUE THREAD

**Issue by thomastai1666** [Mireille-T/The-Bigger-Picture#10]
A user should be able to toggle the setting to automatically generate alt text on page load & see a dropdown with language options (Only English for now)
